industries/views.py

```
@@ -2,16 +2,31 @@
+)
+from .tasks.index_sync import backfill_industry_charts_task
 from companies.serializers import CompanyRankingSerializer
-from industries.serializers import IndustryRankingSerializer
+from industries.serializers import (
+    IndustryRankingSerializer, 
+    IndustryIndexSerializer, 
+    IndustryChartSerializer,
+)
+from news.serializers import IndustryNewsSerializer 
 from news.models import CompanyNews
 from news.serializers import IndustryNewsSerializer
```

Ruff (0.14.13)

Redefinition of unused `IndustryNewsSerializer` from line 29: `IndustryNewsSerializer` redefined here

Remove definition: `IndustryNewsSerializer`

(F811)